core/src/main/java/org/elasticsearch/common/lucene/search/function/FiltersFunctionScoreQuery.java

```diff
@@ -55,18 +55,21 @@ public FilterFunction(Query filter, ScoreFunction function) {
 
         @Override
         public boolean equals(Object o) {
-            if (this == o)
+            if (this == o) {
                 return true;
-            if (o == null || getClass() != o.getClass())
+            }
+            if (o == null || getClass() != o.getClass()) {
                 return false;
+            }
 
             FilterFunction that = (FilterFunction) o;
 
-            if (filter != null ? !filter.equals(that.filter) : that.filter != null)
+            if (filter != null ? !filter.equals(that.filter) : that.filter != null) {
                 return false;
-            if (function != null ? !function.equals(that.function) : that.function != null)
+            }
+            if (function != null ? !function.equals(that.function) : that.function != null) {
                 return false;
-
+            }
```

Comment:
yea it would read better, I only added the missing curly brackets while debugging failures till now, can change that too.